yb-voyager/cmd/importDataFileCommand.go

```diff
@@ -187,12 +188,13 @@ func prepareImportFileTasks() []*ImportFileTask {
 				utils.ErrExit("calculating file size in bytes: %q: %v", filePath, err)
 			}
 			task := &ImportFileTask{
-				ID:           i,
+				ID:           idIndex,
```

Comment:
Why was this ID the same earlier for glob patterns multiple files for a table? May be it was intentional, but have we checked whether there is any change in behavior / UI with this? I can see that this `ID` is used in progress bars:
``` 
pr.progressBars[task.ID] = bar 
pr.totalProgressAmount[task.ID] = totalProgressAmount 
```

Reply:
I thought this was a miss. Let me check the progress bars in this scenario 👍

Reply:
Cool, thanks!